x-pack/test/functional/apps/maps/feature_controls/maps_spaces.ts

```diff
@@ -20,6 +20,13 @@ export default function({ getPageObjects, getService }: KibanaFunctionalTestDefa
   describe('spaces feature controls', () => {
     before(async () => {
       await esArchiver.loadIfNeeded('maps/data');
+      await esArchiver.load('maps/kibana');
```

Comment:
The esArchiver call isn't strictly necessary, but I noticed that the default space was missing when diagnosing these failures. While we don't need the default space for these tests, Kibana assumes that the default space is always available, so I thought it was better to make sure that Kibana was in a known good state before starting this suite.

Reply:
> noticed that the default space was missing when diagnosing these failures

That shouldn't be the case, we explicitly avoid deleting the default space... Perhaps something else is wrong? How can I reproduce locally?

Reply:
I struggle to repro locally, but you can see the error message from jobs where this test has failed: 

https://kibana-ci.elastic.co/job/elastic+kibana+master/JOB=x-pack-ciGroup7,node=immutable/1428/console 

https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/9884/JOB=x-pack-ciGroup7-13,node=immutable/console


My other thought was we were simply unauthorized for the default space, but I would expect to see a different message along the lines of:
> [error][spaces] Unable to navigate to space "default", redirecting to Space Selector. Error: Unauthorized to get default space